Sources/SuperwallKit/Models/Triggers/TriggerResult.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  File.swift
+//  
+//
+//  Created by Yusuf Tör on 22/11/2022.
+//
+
+import Foundation
+
+/// The result of a tracking an event.
+///
+/// Contains the possible cases resulting from tracking an event.
+public typealias TrackResult = TriggerResult
+
+/// The result of a paywall trigger.
+///
+/// Triggers can conditionally show paywalls. Contains the possible cases resulting from the trigger.
+public enum TriggerResult: Sendable, Equatable {
```

Comment:
This won't be accessible from objc

Reply:
This is fine. The only reason it's `public` is because it's used in `getTrackResult`, and `SuperwallEvent`. There's an equivalent Objc version of TrackResult so this doesn't matter